src/data/location.json

```diff
@@ -205,6 +207,7 @@
     "backMarker": {
       "to": 4,
       "position": { "x": 4503, "y": -662, "z": 2055 },
+	  "lookAt": { "x": -175, "y": 305, "z": -4979 },
```

Comment:
Maybe this should point south instead of west since that's the direction of travel